<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>561</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>723</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Shazam</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="MedhatShalaby">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Matching songs</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="song2">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>SELECTED FILE NAME</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSlider" name="slider">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="enabled">
       <number>0</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="openButton2">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" alignment="Qt::AlignRight">
     <widget class="QLabel" name="ratioLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File #1 : File #2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="fileLabel2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
padding-left:5px
}</string>
      </property>
      <property name="text">
       <string>File #2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="fileLabel1">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
padding-left:5px
}</string>
      </property>
      <property name="text">
       <string>File #1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="sliderValue">
      <property name="enabled">
       <number>0</number>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
padding-right:5px
}</string>
      </property>
      <property name="text">
       <string>100 %</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="song1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>SELECTED FILE NAME</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="searchButton">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="openButton1">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" rowspan="3">
     <widget class="QTableWidget" name="table">
      <property name="minimumSize">
       <size>
        <width>395</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>5002</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget{
max-width: 5000px;
}</string>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Similarity Index</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Song Name</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="close1">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="close2">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
